icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc

```diff
@@ -260,9 +260,9 @@ sbn::crt::CRTHit CRTHitRecoAlg::FillCRTHit(vector<uint8_t> tfeb_id, map<uint8_t,
     crtHit.pesmap      = tpesmap;
     crtHit.peshit      = peshit;
     crtHit.ts0_s_corr  = time0 / 1'000'000'000; 
-    crtHit.ts0_ns      = time0;
+    crtHit.ts0_ns      = time0 % 1'000'000'000;
     crtHit.ts0_ns_corr = time0; 
-    crtHit.ts1_ns      = time1;
+    crtHit.ts1_ns      = time1 % 1'000'000'000;
```

Comment:
I'm trying to understand this better and I'm guessing I'm missing background information about the construction of the timestamps. 

Are the ts0 and ts1 represented as second+nanosecond timestamps and you are trying to only extract the last nine digits from the nanosecond component in this effort? This what this change represents to me and I wanted to understand if I have the interpretation correct

Reply:
That is correct. `ts0` is a 19-digit  integer which represent the time stamp in nanosecond, and we are trying to save the last nine digits-only to `ts1_ns`, and the second-only part (the rest 10 digits) to `ts0_s`. It looks to me that this was the original `intention` of introducing the lines https://github.com/SBNSoftware/sbnobj/blob/v09_13_12/sbnobj/Common/CRT/CRTHit.hh#L49-L50

Reply:
Thanks Jaesung, I thought that was the case. Thanks for confirming.